### ISSUE TITLE
[CSS] Implement the CSS Text 4 property `hyphenate-character`

### ISSUE THREAD

**Issue by Rumyra** [mdn/content#11596]
## Acceptance Criteria

- [ ] The listed features are documented sufficiently on MDN
- [ ] BCD is updated
- [ ] Interactive example and data repos are updated if appropriate
- [ ] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [ ] Set bugs to `dev-doc-complete`
- [ ] Add entry to Firefox release notes if feature is enabled in release
or
- [ ] Add entry to Firefox experimental features page if feature is not yet enabled in release

## Features to document

https://bugzilla.mozilla.org/show_bug.cgi?id=1746187

**Comment by hamishwillee** [comment]
@Rumyra Can you help or advise on the data section for this? It doesn't exist yet ; https://github.com/mdn/data/blob/main/css/properties.json

There are just two properties, so it looks like either of these two:
```
hyphenate-character: auto;
hyphenate-character: <string>;
```
Effectively I guess if you don't specify this it will choose auto. 

There is also a webkit prefix version that is used by Chromium and Safari

**Comment by hamishwillee** [comment]
@Rumyra Not in release because feature behind preference

Status

- [ ] Docs - #11875
- [ ] BCD - https://github.com/mdn/browser-compat-data/pull/14488 
- [ ] Experimental features
- [ ] dev-doc-complete

**Comment by Rumyra** [comment]
> @Rumyra Can you help or advise on the data section for this? It doesn't exist yet ;

Yes I'll go through the whole release project & update mdn/data with all the things 👍

**Comment by Rumyra** [comment]
Just moving back because of the bcd review

**Comment by Rumyra** [comment]
FYI https://github.com/mdn/data/pull/544
